### PR TITLE
Chat window

### PR DESCRIPTION
**Pull request by vladbyk89**

### CODE REVIEW

**Review by tomer6633 — APPROVED** [review]
nice start but when i create a chat and i enter to it i go to the menu chats and see the fake DB
i would like to know why 
but good job

**Comment by vladbyk89** [comment]
Can you be more specific @tomer6633? I can't really understand what you mean.

File: client/app/(tabs)/_layout.tsx [hunk 1]
```diff
@@ -8,35 +8,61 @@ import CustomHeader from '../../components/CustomHeader';
...
-          header: () => <CustomHeader />,
-          tabBarIcon: ({ color, focused }) => <MaterialCommunityIcons name={focused?"bell":"bell-outline"} size={focused?32:24} color={color} style={{color:focused?"lightblue":color}}/>,
-        }}
-      />
-    </Tabs>
-  );
+    return (
+        <Tabs
+            screenOptions={{
+                tabBarActiveTintColor: Colors[colorScheme ?? 'light'].tint,
```
Comment (itayz1e) [inline client/app/(tabs)/_layout.tsx:14]: What is it supposed to do?

File: client/app/(tabs)/_layout.tsx [hunk 2]
```diff
@@ -8,35 +8,61 @@ import CustomHeader from '../../components/CustomHeader';
...
+                    tabBarIcon: ({ color, focused }) => (
+                        <Ionicons
+                            name={
+                                focused
+                                    ? 'chatbubble-ellipses'
+                                    : 'chatbubble-ellipses-outline'
+                            }
+                            size={focused ? 32 : 24}
+                            color={color}
+                            style={{ color: focused ? 'lightblue' : color }}
```
Comment (itayz1e) [inline client/app/(tabs)/_layout.tsx:31]: Can it sit inside the style file?
It would be more professional then...!

File: client/app/chat.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+// expo
+import { useLocalSearchParams } from 'expo-router';
+
+// zustand store
+import useChats from '@/store/useChats';
+
+interface MessageType {
+    text: string;
+    user: string;
+}
```
Comment (itayz1e) [inline client/app/chat.tsx:22]: You can't put it in DBmocke and import it from there?

**Review by itayz1e — COMMENTED** [review]
Nice work keep it up!  😎

**Pull request merged**